docs/source/install.rst

```diff
@@ -323,6 +336,19 @@ C flags for compilation of the X-PSI extensions. Because the library location
 will not change for runtime, we state the runtime linking instructions at
 compilation in the ``setup.py`` script.
 
+To check whether installation proceeded correctly and the software is functioning as expected,
+execute the following:
+
+.. code-block:: bash
+
+    cd examples/examples_fast/Modules/
+    python main.py
+
+This module performs a ``likelihood check``. If the likelihood value calculated matches
+the given value, X-PSI is functioning as expected, else it will raise an error message.
+The module will then initiate sampling using MultiNest (assuming that it's installed),
+and given the settings, it should take ~15 minutes. To cancel mid-way press ``ctrl + C``.
```

Comment:
In the end, it seems that the test-run takes now only 5 minutes! Can you modify accordingly?

Reply:
Done